docs/src/dev/developer/release.asciidoc

```diff
@@ -242,12 +242,15 @@ for generating javadoc.
 .. `mvn deploy -pl :gremlin-dotnet-source -DskipTests -Dnuget`
 .. `mvn deploy -pl gremlin-javascript -DskipTests -Dnpm`
 .. `mvn deploy -pl gremlint -DskipTests -Dnpm`
+.. `git tag gremlin-go/v3.5.4`
+.. `git push --tags`
 . Review the GLV releases
 .. link:https://pypi.org/project/gremlinpython/[gremlin-python - PyPi]
 .. link:https://www.nuget.org/packages/Gremlin.Net/[Gremlin.Net - nuget]
 .. link:https://www.nuget.org/packages/Gremlin.Net.Template/[Gremlin.Net.Template - nuget]
 .. link:https://www.npmjs.com/package/gremlin[gremlin - npm]
 .. link:https://www.npmjs.com/package/gremlint[gremlint - npm]
+.. link:https://pkg.go.dev/github.com/apache/tinkerpop/gremlin-go/v3/driver/[gremlin-go - go packages]
```

Comment:
Add 'v3' to hyperlink

Reply:
fixed in https://github.com/lyndonbauto/tinkerpop/pull/83/commits/1001ef64e2ae1df6ee8813ea5dbeac2f4bff0a19